packages/core/src/RenderingEngine/BaseVolumeViewport.ts

```diff
@@ -312,6 +409,17 @@ abstract class BaseVolumeViewport extends Viewport implements IVolumeViewport {
         this.id,
         suppressEvents
       );
+      if (i === 0 && !this.voiRange) {
+        const voiRange = actor
+          .getProperty()
+          .getRGBTransferFunction(0)
+          // @ts-ignore: vtk d ts problem
+          .getRange();
+        this.voiRange = {
+          lower: voiRange[0],
+          upper: voiRange[1],
+        };
```

Comment:
we basically do this inside `createVolumeActor`, how is it different?

Reply:
I don't think the voiRange is saved to the volume anywhere, here we set the voiRange on the `BaseVolumeViewport` instance. I guess this could be moved into create volume actor, but it would require passing `this` into it.

Reply:
I guess one way to do this is to have BaseVolumeViewport implement a getter for `voiRange`. This getter takes in an index for the volume actor of interest with default being 0. A VOIRange object is returned.

Thoughts on this?

Reply:
I would say let's take out the `triggerVOIModified` from inside the createVolumeActor, and put it inside the base